foundation_cms/static/js/blocks/impact_numbers_block.js

```diff
@@ -0,0 +1,130 @@
+/**
```

Comment:
Short explanation of the JS file:

Takes whatever numerical value is set for the Impact Number, and randomizes each digit for 1.5 seconds before landing on the actual value.

Reply:
Thanks @mmmavis! I was able to implement something close to the codepen provided and its ready for re-review 👍